src/test/java/com/github/dockerjava/client/AbstractDockerClientTest.java

```diff
@@ -180,13 +183,13 @@ public static void assertContainerHasVolumes(InspectContainerResponse inspectCon
             Volume... expectedVolumes) {
```

Comment:
did you know about custom hamcrest matchers?

you can use form `assertThat(inspectContainerResponse, hasMounts(containsInAnyOrder(expectedVolumes)))`

with simple `FeatureMatcher`:

```java
new FeatureMatcher<InspectContainerResponse, List<Volume>>(anotherMatcherForChild, "expected comment", "actual comment") {
    @Override
     public List<Volume> featureValueOf(InspectContainerResponse item) {
         // ...
          for (Mount mount : item.mounts) {
                volumes.add(mount.getDestination());
         }
         return volumes;
     } 
}
```

it copies assertion logic inside of method of object. The main advantage - you can combine such checks:
`assertThat(inspectContainerResponse, not(hasMounts(containsInAnyOrder(expectedVolumes))))`

`assertThat(inspectContainerResponse, allOf(hasMounts(containsInAnyOrder(expectedVolumes)), hasIP("127.0.0.1")))`

and so on.

More info here: http://hamcrest.org/JavaHamcrest/